recipe/conda_forge_ci_setup/feedstock_outputs.py

```diff
@@ -14,6 +17,29 @@
 STAGING = "cf-staging"
 
 
+@lru_cache(maxsize=1)
+def _load_allowed_autoreg_feedstock_globs(time_int):
+    r = requests.get(
+        "https://raw.githubusercontent.com/conda-forge/feedstock-outputs/"
+        "main/feedstock_outputs_autoreg_allowlist.yml"
+    )
+    r.raise_for_status()
+    yaml = YAML(typ="safe")
+    return yaml.load(r.text)
+
+
+def load_allowed_autoreg_feedstock_globs():
+    return _load_allowed_autoreg_feedstock_globs(time.monotonic() // 120)
+
+
+def check_allowed_autoreg_feedstock_globs(feedstock, output):
+    fs_pats = load_allowed_autoreg_feedstock_globs()
+    for pat in fs_pats.get(feedstock, []):
+        if fnmatch(output, pat):
+            return True
+    return False
```

Comment:
Shouldn't these go in https://github.com/conda-forge/conda-forge-metadata/blob/main/conda_forge_metadata/feedstock_outputs.py ?

Reply:
Sure, I can move them there.